Final touches

vnajack opened the pull request:
- moved files to improve directory organization
- added max time for avatar to NOT blink (don't want their little eyes to get dry)
- polished README and added links to documentation, screenshots, and Adobe Illustrator script
- added error explanations for when Rhubarb and ffmpeg are not installed correctly
- added links to example output videos
- closes #297 
- closes #290 
- closes #269 
- closes #242

vnajack committed:
Final touches

- moved files to improve directory organization
- added max time for avatar to NOT blink (don't want their little eyes to get dry)
- polished README and added links to documentation, screenshots, and Adobe Illustrator script

mkuelker reviewed (approved):
Looks good! Nice work!

mkuelker reviewed (approved):
Looks good! Nice work!

mkuelker reviewed (approved):
Looks good! Nice work!

vnajack committed:
Error codes and example videos

- added error information in case Rhubarb and ffmpeg aren't installed correctly
- included example videos in the README to replace the gif

vnajack committed:
fix for video not showing in readme

- linked to videos because the video embed didn't work even though it previewed fine on my device
- also kept the gif

edouarde1 commented on README.md:
```diff
@@ -148,14 +151,18 @@ could lead to lip synchronization errors.
 | 403 Forbidden | Request is recognized by the server but refused. Likely due to connecting to the wrong port |
 | 500 Internal Server Error | The API has not been configured properly on the host server machine |
 
+If Rhubarb is not installed correctly or fails to read the audio and text files, the output may include a file not found error for `./rhubarb`, an `exit code (status): null`, and a file not found error for the .MP4 output file. Please ensure Rhubarb is installed correctly (refer to [setup](#setup)), check that the audio file is a valid .WAV file, and that the text file is a valid .txt file.
```
Great! this is really good that we document this :)

edouarde1 commented on README.md:
```diff
@@ -203,11 +210,19 @@ The [frame data output](#frame-data-output) includes a path to the image and the
 output. Once the video file is created, it is then returned to the user as a response from the API.
 
-## Example
+## Output Examples
```
sweet!!

edouarde1 reviewed (approved):
great additions to the documentation with the error codes and output contents
